### PR TITLE
[smclient] Add SANtricity storage device (NetApp) support

### PR DESCRIPTION
**Pull request by vikas-goel**
Signed-off-by: Vikas Goel <vikas.goel@gmail.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

**Comment by vikas-goel** [comment]
Check failures for unrelated issues.

File: sos/report/plugins/smclient.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            "save storageArray supportData file=\"supportbundle\";",
+        ]
+
+        logpath = self.get_cmd_output_path(make=False)
+
+        for ssname in ssnames:
+            self.add_cmd_output([
+                "SMcli localhost -n %s -c '%s'" % (ssname, subcmd)
+                for subcmd in subcmds
+            ], runat=logpath, timeout=450)
```
Comment (TurboTurtle) [inline sos/report/plugins/smclient.py:62]: I'm assuming this generates some file in the working directory, and that the filename would be `supportbundle`?

If so, my first question would be is "supportbundle" the term used by the team(s)/user(s) that look at this data?

Secondly, if there are more than 1 entities in `ssnames`, would this not overwrite the `supportbundle` file, or is `SMcli` appending this data to that file if it already exists?

**Review by TurboTurtle — COMMENTED** [review]
Overall this looks good to me. Just need a clarification on the actions of the `debug` plugin option.

File: sos/report/plugins/smclient.py (hunk 1 above)
Comment (vikas-goel) [reply sos/report/plugins/smclient.py]: I have updated the code to generate a unique filename. It is going to look like `support-myhost_rbod_nbj089.7z`. The file is generated by the SMcli command in the current working directory. Hence, the `logat` is needed.

As you said, the user will use the file to look at the data to troubleshooting issues.

Thank you for the feedback.

**Review by TurboTurtle — APPROVED** [review]

**Comment by vikas-goel** [comment]
@pmoravec , please review

**Review by pmoravec — APPROVED** [review]

**Pull request merged**